### ISSUE TITLE
login method requires captcha now

### ISSUE THREAD

**Issue by HouZzzz** [lvoegl/ScIdClient#4]

**Comment by lvoegl** [comment]
I can't reproduce this issue with my setup. What requests are you sending exactly?

**Comment by HouZzzz** [comment]
![image](https://github.com/user-attachments/assets/6d6b50d9-57c2-45f9-a861-b3bdf5b4a119)

`    headers = {
        "User-Agent": f"scid/4543 (Android; laser-prod)",
        "Authorization": "",
    }
    response = requests.post(url="https://id.supercell.com/api/ingame/account/login", data={
        "email": body['email'],
        "remember": 'true',
        "lang": "en",
        "game": "laser",
        "env": "prod"
    }, headers=headers)`
it was working steadily until ~August 15. After that it was responding with status 200, but the codes stopped coming to the mail. also in the game and on the supercell id site login page appeared the text: “protected by Captcha”

**Comment by lvoegl** [comment]
I have also experienced issues with codes not being sent, although this is not an issue with this client since the response is code 200 `OK` as you said. I still don't see where captchas would be a problem here.

**Comment by HouZzzz** [comment]
I also tried to do it on different hosts, but the result is the same. Code 200, but no code in the mail. 0 codes sent from ~50 attempts since ~August 15th. I checked questions about this on different forums, and yes, other people had this problem too, this endpoint can't send codes anymore. BUT, maybe it's because all my hosts are in Russia. I will try some hosting in EU and let you know the result. The fact that it works for you encourages you to make a decision, thank you!
